content/src/content/jcr_root/apps/core/wcm/components/contentfragmentlist/v1/contentfragmentlist/_cq_dialog/.content.xml

```diff
@@ -92,6 +92,27 @@
                             </column>
                         </items>
                     </elements>
+                    <results
```

Comment:
I would add the configuration to the `Properties` tab. Having a new tab with only one property looks weird.

Reply:
Probably makes best sense below the current `Tags` field (below filters, as in the List component)